pymeasure/display/widgets/plot_frame.py

```diff
@@ -70,7 +70,7 @@ def _setup_ui(self):
 
         self.plot = self.plot_widget.getPlotItem()
 
-        style = dict(self.LABEL_STYLE | {'justify': 'right'})
+        style = dict({'justify': 'right'}, **self.LABEL_STYLE)
```

Comment:
This is a very minor thingy: changing the order causes values in `self.LABEL_STYLE` to take precedence over the new values, so maybe this would be a better order:
```suggestion
        style = dict(self.LABEL_STYLE, justify='right')
```

Reply:
Sounds good, fixed in last commit.

Reply:
@CasperSchippers the last runner seems to have borked, can you restart it?

Reply:
I have re-run failed testa and now it's ok